libcalico-go/lib/net/mac.go

```diff
@@ -26,12 +27,14 @@ type MAC struct {
 
 // MarshalJSON interface for a MAC
 func (m MAC) MarshalJSON() ([]byte, error) {
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
Would it make sense to have a libcalico-go package to wrap this up?

```go
package json

jsoniter "github.com/json-iterator/go"


func Marshal(s string) []byte, error {
  return jsoniter.ConfigCompatibleWithStandardLibrary.Marshal(s)
}
```

Would turn this into an import change rather than an import + local variable change. Unless you think there's a future where we want to use different jsoniter implementations in different places?